scripts/postinst

```diff
@@ -1,7 +1,16 @@
 #!/bin/sh
 
 set -e
-Python_SITELIB=$(python3 -c "from distutils import sysconfig;print(sysconfig.get_python_lib(plat_specific=False,standard_lib=False))")
+Python_SITELIB=$(python3 <<EOF
+import sys
+if sys.version_info < (3, 10, 0):
+    from distutils import sysconfig
+    print(sysconfig.get_python_lib(plat_specific=False,standard_lib=False))
+else:
+    from sysconfig import get_path; print(get_path('platlib', 'deb_system'))
```

Comment:
Do you know where the `deb_system` scheme is documented?

Reply:
No :(
Here is where I got that from: https://forum.freecad.org/viewtopic.php?t=67985
Which links to this "interesting" article: https://ffy00.github.io/blog/02-python-debian-and-the-install-locations/
Add this to the many things I hate about Python.